config/default.schema.json

```diff
@@ -1578,23 +1586,6 @@
 				"required": ["FEATURE_USER_MIGRATION_SYSTEM_ID"]
 			}
 		},
-		"ANTIVIRUS": {
-			"if": {
-				"properties": {
-					"ENABLE_FILE_SECURITY_CHECK": {
-						"const": true
-					}
-				}
-			},
-			"then": {
-				"required": [
-					"FILE_SECURITY_CHECK_SERVICE_URI",
-					"FILE_SECURITY_SERVICE_USERNAME",
-					"FILE_SECURITY_SERVICE_PASSWORD",
-					"API_HOST"
-				]
-			}
-		},
 		"LERNSTORE_ENABLED": {
```

Comment:
can be removed as well since ES_USER and ES_PASSWORD is per default an empty string, thus this never fails, also meaning that you can remove LERNSTORE_ENABLED from your configmap "overwrites"